solvis/geometry.py

```diff
@@ -155,26 +216,54 @@ def refine_dip_direction(point_a: Point, point_b: Point, original_direction: flo
 
 def dip_direction(point_a: Point, point_b: Point) -> float:
     """
-    Computes the dip_direction in degrees from the points A & B
+    Computes the dip direction for a strike from Point A to Point B.
+
+    Parameters:
+        point_a: the first point
+        point_b: the second point
 
-    :param point_a: the first point
-    :param point_b: the second point
+    Returns:
+        the dip direction in degrees
     """
     dip_dir = strike(point_a, point_b) + 90
     return dip_dir + 360 if dip_dir < 0 else dip_dir
 
 
-def circle_polygon(radius_m: int, lat: float, lon: float):
+def circle_polygon(radius_m: int, lat: float, lon: float) -> Polygon:
```

Comment:
@DrCuriosity  should we refer to these external types explicity i.e. `shapely.Polygon` ? it would make it clearer to the  user that they'll also need to know about shapely.

Reply:
I have some docs at the beginning on the file, but we could make it more explicit in the signatures.

> Functions for working with Shapely geometries.
> 
> These functions require [Shapely](https://shapely.readthedocs.io/en/stable/index.html) to be installed.

Reply:
Yeah I think it's good practice  ... it makes the functions more grokable IMHO

Reply:
Changing to `annotations_path: full` gets us this without any further signature changes in the code:
![image](https://github.com/GNS-Science/solvis/assets/8766466/0d2309e4-2417-47e9-9d4c-a030bef57d67)

Does that suit?

Reply:
It doesn't address the python shell help, but let's go with this for now - thanks.  It's certainly a good solution in the web docs.